src/include/miopen/execution_context.hpp

```diff
@@ -58,6 +58,7 @@ class rocm_meta_version
 
 namespace miopen {
 
+const char* const MIOPEN_PERFDB_SCHEMA_VER = "1.0.0";
```

Comment:
This defines the global pointer and C-string in each module where this header is included. Perhaps modern C++ "optimizations" can optimize this out, but this... I do not believe that we want this.

Reply:
It can be in the header but it should be moved to the class level so it will point to the same address across all TUs.

Reply:
Moved to a  static member